bench_test.go

```diff
@@ -172,13 +186,13 @@ func BenchmarkInject_TextMap_100BaggageItems(b *testing.B) {
 	benchmarkInject(b, opentracing.TextMap, 100)
 }
 
-func BenchmarkInject_Binary_Empty(b *testing.B) {
-	benchmarkInject(b, opentracing.Binary, 0)
-}
-
-func BenchmarkInject_Binary_100BaggageItems(b *testing.B) {
-	benchmarkInject(b, opentracing.Binary, 100)
-}
+// func BenchmarkInject_Binary_Empty(b *testing.B) {
```

Comment:
intentional?

Reply:
we don't support binary yet. Let's reintroduce it when it is supported.